cloudflared/rootfs/etc/s6-overlay/scripts/cloudflared-config.sh

```diff
@@ -306,34 +306,6 @@ createDNS() {
     fi
 }
 
-# ------------------------------------------------------------------------------
-# Set Cloudflared log level
-# ------------------------------------------------------------------------------
-setCloudflaredLogLevel() {
-local log
-
-# Map Home Assistant log levels to Cloudflared
-if bashio::config.exists 'log_level' ; then
-    case $(bashio::config 'log_level') in
-        "trace") log="info";;
-        "debug") log="info";;
-        "info") log="info";;
-        "notice") log="info";;
-        "warning") log="warn";;
-        "error") log="error";;
-        "fatal") log="fatal";;
-    esac
-else
-    log="info"
-fi
-
-# Write log level to S6 environment
-printf "%s" "${log}" > /var/run/s6/container_environment/CLOUDFLARED_LOG
-CLOUDFLARED_LOG=${log}
```

Comment:
We have to set this variable (CLOUDFLARED_LOG) somewhere else in the script from the run options of cloudflared if present in order for all the runs within this script (DNS creation, ...) to use the correct log-level.
Another option would be to remove the loglevel from the runs in here, which is probably ok but in the end not 100% correct.

Reply:
You are right, I first thought about removing it completely, but I implemented a logic to read the log level from the run_parameters, if set by the user. 
If you have any ideas on how to simplify the logic, please let me know.